sensu/plugins/check-network-traffic.rb

```diff
@@ -0,0 +1,65 @@
+#!/opt/sensu/embedded/bin/ruby
+#
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-plugin/check/cli'
+
+class CheckNetworkStats < Sensu::Plugin::Check::CLI
+
+  linkspeed = `ethtool eth0 | grep Speed`
+  puts linkspeed
```

Comment:
I removed line 11, because it's unnecessary to print the link speed.  I was using it to debug for a bit, and forgot to remove it.